sound/soc/sof/intel/hda.h

```diff
@@ -447,6 +447,9 @@ struct sof_intel_hda_dev {
 
 	/* sdw context allocated by SoundWire driver */
 	struct sdw_intel_ctx *sdw;
+
+	/* FW clock config, 0:HPRO, 1:LPRO */
+	u32 clk_config;
```

Comment:
why not call if clk_config_lpro? its more intuitive that FW is configured for LPRO and not otherwise